codesydney/woc
# [Add] install nodemon to dev depedency

## Rationale

fix the issue of not able to run 'npm start'

## Advice for Reviewers & Testing Notes

- How to reproduce
  *  make sure you are on main branch
  *  run 'npm run dev-server' from root folder
  *  you should see  
  ```
    > server@1.0.0 start /Users/md/Documents/codes/woc/server
    > nodemon

    sh: nodemon: command not found
    npm ERR! code ELIFECYCLE
    npm ERR! syscall spawn
    npm ERR! file sh
    npm ERR! errno ENOENT
    npm ERR! server@1.0.0 start: `nodemon`
    npm ERR! spawn ENOENT
    npm ERR! 
    npm ERR! Failed at the server@1.0.0 start script.
    npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
  ```
- How to test
  * make sure you are on the branch 'debug-nodemon'
  * npm install
  *  make sure you are on the root folder
  *  run command 'npm run dev-server' 
  *  you should be able to see server running msg, something like below 
  ````
      ➜  woc git:(debug-nodemon) npm run dev-server
  
      > woc@1.0.0 dev-server /Users/md/Documents/codes/woc
      > cd server && npm run start
      
      
      > server@1.0.0 start /Users/md/Documents/codes/woc/server
      > nodemon
      
      [nodemon] 2.0.7
      [nodemon] to restart at any time, enter `rs`
      [nodemon] watching path(s): *.*
      [nodemon] watching extensions: js,mjs,json
      [nodemon] starting `node server.js`
      [2021-04-14T13:37:09.006Z] [INFO] [SERVER] Server running on localhost:8080

  ````

## Screenshots:

- n/a

## Task Name

- Issue Title #22 


## Linting Checklist
